cmd/nerdctl/container/container_restart.go

```diff
@@ -38,6 +38,7 @@ func NewRestartCommand() *cobra.Command {
 		SilenceErrors:     true,
 	}
 	restartCommand.Flags().UintP("time", "t", 10, "Seconds to wait for stop before killing it")
+	restartCommand.Flags().StringP("signal", "s", "", "Signal to send to stop the container, before killing it")
```

Comment:
The docs needs to be updated too
https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md#whale-nerdctl-restart